src/app/modules/dealers/models/dealers.entity.ts

```diff
@@ -0,0 +1,65 @@
+import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity()
+export class Dealers {
+
+    @PrimaryGeneratedColumn()
+    id: string;
+
+    @Column()
+    name: string;
+
+    @Column({default: ''})
```

Comment:
naming convention mismatch. I'll check linting, it should catch this type of thing.